hugr/Cargo.toml

```diff
@@ -34,7 +36,11 @@ rstest = { workspace = true }
 lazy_static = { workspace = true }
 criterion = { workspace = true, features = ["html_reports"] }
 serde_json = { workspace = true }
+bumpalo = { workspace = true, features = ["collections"] }
 
 [[bench]]
 name = "bench_main"
 harness = false
+
+[profile.bench]
```

Comment:
That includes debug symbols into the benchmark binaries, which is very useful for profiling. I could take it out and put it in locally when I need it for flamegraphs, or leave it in.

Reply:
It isn't debug mode (as opposed to release mode), but whether to include debug information (like function names) in the compiled binary. It might have a small impact on performance due to locality since the binary is consequently larger.